packages/trousers.rb

```diff
@@ -27,11 +27,15 @@ class Trousers < Package
 
   def self.build
     system './configure', '--disable-static', '--with-gui=none',
-      "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"
+      "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}", 'CFLAGS=-fcommon'
```

Comment:
Do we need new binaries?

Reply:
Strictly speaking, I don't think they are required by this change, but I feel like now is as good a time as any to rebuild so that this is linking against openssl instead of libressl. After all, trying to `crew reinstall -s` is how I ran into this issue in the first place! 😆

Reply:
Since this is a core package, we'll need to create binaries anyway.  @satmandu: Did you want to provide new binaries?

Reply:
Sure. Does CREW_OPTIONS not work for this?

Reply:
Going to try rebooting my system to get these uploaded. But I've copied in the URLs and hashes for when this upload succeeds.